cpp/tests/io/parquet_test.cpp

```diff
@@ -580,18 +580,21 @@ TEST_F(ParquetWriterTest, Strings)
   expected_metadata.column_metadata[1].set_name("col_string");
   expected_metadata.column_metadata[2].set_name("col_another");
 
-  auto filepath = temp_env->get_temp_filepath("Strings.parquet");
   cudf_io::parquet_writer_options out_opts =
-    cudf_io::parquet_writer_options::builder(cudf_io::sink_info{filepath}, expected->view())
-      .metadata(&expected_metadata);
+    cudf_io::parquet_writer_options::builder(
+      cudf_io::sink_info(std::vector<std::string>{"first.parquet", "second.parquet"}),
+      expected->view())
+      .partitions({{1, 2}, {4, 1}})
+      .metadata(&expected_metadata)
+      .stats_level(cudf::io::STATISTICS_NONE);
```

Comment:
is this a limitation of partitioned writes?

Reply:
It's not. This test was changed just for debugging.